examples/pset_blind_coinjoin.rs

```diff
@@ -287,3 +285,43 @@ fn main() {
     tx.verify_tx_amt_proofs(&secp, &[btc_txout, asset_txout])
         .unwrap();
 }
+
+
+/// Xorshift
+pub struct CrappyRng(u64);
+
+impl CrappyRng {
+    fn new(initial: core::num::NonZeroU64) -> Self {
+        Self(initial.get())
+    }
+}
+
+impl rand::RngCore for CrappyRng {
+
+    fn next_u32(&mut self) -> u32 {
+        self.next_u64() as u32
+    }
+
+    fn next_u64(&mut self) -> u64 {
+        let mut x = self.0;
+        x ^= x << 13;
+        x ^= x >> 7;
+        x ^= x << 17;
+        self.0 = x;
+        x
+      }
+
+    fn fill_bytes(&mut self, dest: &mut [u8]) {
+        for chunk in dest.chunks_mut(8) {
+            let x = self.next_u64().to_be_bytes();
+            chunk.copy_from_slice(&x[..chunk.len()]);
+
+        }
+    }
+
+    fn try_fill_bytes(&mut self, dest: &mut [u8]) -> Result<(), rand::Error> {
+        Ok(self.fill_bytes(dest))
```

Comment:
This code is duplicated in the examples, I thought it to be better to duplicate the code instead of having it in the main lib

Reply:
From the examples, I cannot access something I put behind a `#[cfg(test)] ` in the lib.

eg.
If I put `#[cfg(test)] pub const CIAO: u8 = 0u8;` in `src/lib.rs`

and I do `cargo run --example pset_blind_coinjoin`

I got `error[E0425]: cannot find value `CIAO` in crate `elements``

Reply:
Oh, derp, I missed that this was in examples rather than unit tests.

Now I'm unsure if we should be using a broken RNG given that this is "example" code ... we should probably go back to using `rng_chacha` :P

Reply:
Ok, but the wrong interpretation due the ambivalent nature of this examples/tests leave me the doubt of moving those in `/tests`

Reply:
I think for now let's just go with this, since it's blocking our move to `rand 0.8`. We can rearrange things later.

Reply:
Opened issue #149